### PR TITLE
Lorawan juzi

### PR DESCRIPTION
**Pull request by julizet**

### CODE REVIEW

File: source/projects/lorawan/en/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+//*** Werte bekommt Ihr aus der TTN-Console, Format msb! ***
+static u1_t NWKSKEY[16] = {
+    0xB2, 0x5F, 0x35, 0x64, 0xB4, 0x89, 0xB8, 0x09, 0x08, 0x12, 0x7D, 0xAC, 0x0F, 0xC6, 0xF1, 0x5C
+    }; 
+    
+static u1_t APPSKEY[16] = {
+    0xB2, 0x5B, 0x16, 0x81, 0x53, 0x70, 0x49, 0xBF, 0x24, 0xBD, 0x55, 0xB2, 0xB5, 0xF6, 0xCB, 0x46
+    }; 
+
+// ACHTUNG: DEVICE ADRESSE MIT PRÄFIX 0x
```
Comment (fabianmoronzirfas) [inline source/projects/lorawan/en/index.md]: Code comments should in english…

File: source/projects/lorawan/en/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+Once the device is registered, you can get **the aforementioned three variables** (device address, network session key and app key) from the TTN console. You can find the Sketch (i.e., the program code) [here on GitHub](https://github.com/technologiestiftung/werkstatt/tree/master/codes_sketches) – name: *GPSTracker_LoRa_Nano_V2.ino*
+
+The following lines need to be adjusted in the code:
+
+```js
+//*** Werte bekommt Ihr aus der TTN-Console, Format msb! ***
+static u1_t NWKSKEY[16] = {
+    0xB2, 0x5F, 0x35, 0x64, 0xB4, 0x89, 0xB8, 0x09, 0x08, 0x12, 0x7D, 0xAC, 0x0F, 0xC6, 0xF1, 0x5C
```
Comment (fabianmoronzirfas) [inline source/projects/lorawan/en/index.md]: Would look a little nicer if you make blocks like 

```c
0xB2, 0x5F,
0x35, 0x64,
0xB4, 0x89,
0xB8, 0x09,
//…
```

File: source/projects/lorawan/en/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+## Areas of application
+
+Anyone now wondering why we still pay for mobile network & internet when there seems to be a perfectly good alternative – and free – network available is unfortunately going to be disappointed: the **data transfer rate of the LoRaWan only ranges from 292 bit/s to a maximum of 51 kBit/s**. When using the LMIC library, which we used in our example code below to transfer data, 13 bytes of the maximum 51 bytes are reserved for so-called overhead data, i.e. metadata. The maximum data packets are therefore much smaller compared to Ethernet (easily up to 1,500 MBit/s) or the internet (100 MBit/s usual for home use), which is why LoRaWan is primarily suitable for the use of various **IoT applications and sensors**.
+
+{% include macro-image-section-markdown-small.html src="../images/lorawan-use-cases.jpg" caption="LoRaWan Use Cases (© The Things Network)"%}
+
+<!-- <img src="../images/lorawan-use-cases.jpg" style="margin-left:6%; margin-right:4%; float:right; width:70%" alt="LoRaWan Use Cases"> -->
```
Comment (fabianmoronzirfas) [inline source/projects/lorawan/en/index.md]: Remove unused code

File: source/projects/lorawan/en/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+status: finished
+date: 2019-09-30
+
+materialsIncluded:
+  - name: "GitHub"
+    link: "https://github.com/technologiestiftung/werkstatt"
+
+redirect_from:
+  - /projects/LORAWAN/index_en.html
+  -
```
Comment (fabianmoronzirfas) [inline source/projects/lorawan/en/index.md]: remove empty line

File: source/projects/lorawan/de/index.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+Die letzten zwei Workshops der offenen Werkstatt im [CityLAB Berlin](https://www.citylab-berlin.org/) widmeten sich dem Thema **GPS Tracking mit LoRaWan**. Dabei wurden die Teilnehmer\*innen mit entsprechender Hard- und Software durch unsere [Bauanleitung auf GitHub](https://github.com/technologiestiftung/werkstatt) geführt und mit Hilfe von LoRa-Nodes, aber auch kleineren Bauteilen wie der [Dragino LoRaBee](https://www.dragino.com/products/lora/item/109-lora-bee.html), Daten über das LoRa-Netz übertragen. 
+
+Im letzten Workshop haben wir einen optimierten Tracker gebaut. Entgegen zu den LoRa-Nodes, wurden die individuell Bauteile zusammengestellt, sodass wir am Ende eine sehr robuste Variante (gelötet) mit sehr kleinem Packmaß gebaut haben.
+
+
+### Ein kleiner Vorgeschmack
+{% include macro-image-section-markdown-small.html src="../images/Nano_Xbee_Image.png" caption="GPS-LoRa-Tracker, wie er bei der Offenen Werkstatt gebaut wurde (© TSB)"%}
+
+<!-- <img src="../images/Nano_Xbee_Image.png" style="margin-left:6%; margin-right:4%; float:right; width:60%" alt="GPS-LoRa-Tracker CityLAB"> -->
```
Comment (fabianmoronzirfas) [inline source/projects/lorawan/de/index.md]: Same here for the unused code

File: source/projects/lorawan/en/index.md (hunk 1 above)
Comment (julizet) [reply source/projects/lorawan/en/index.md]: translted all the german code comments for english version

File: source/projects/lorawan/en/index.md (hunk 2 above)
Comment (julizet) [reply source/projects/lorawan/en/index.md]: alright, changed to groups of four

File: source/projects/lorawan/de/index.md (hunk 5 above)
Comment (julizet) [reply source/projects/lorawan/de/index.md]: removed comment code

**Pull request merged**